### PR TITLE
chore: add Umar Zaki Gunawan as 2024 Academician

### PR DESCRIPTION
**Pull request by marzkigun27**
Just add my name, and my Github link into Readme.md

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician.
 - [Mohammad Fiqri Firmansyah](https://github.com/TakanashaTaryu) [RYU]
   > As an academician.
+-[Umar Zaki Gunawan](https://github.com/marzkigun27). [UZY]
```
Comment (fakhrip) [inline Readme.md:76]: Wrong format, look again carefully, and please fix it, thanks

**Review by fakhrip — CHANGES_REQUESTED** [review]
Congrats for the first PR, but there are something you have to fix first, ask again for my review after you pushed the changes, thanks

File: Readme.md (hunk 1 above)
Comment (marzkigun27) [reply Readme.md]: forgive my lack of attention, I've already corrected it, please check.

**Review by fakhrip — APPROVED** [review]
Okay cool, good job!

File: Readme.md (hunk 1 above)
Comment (fakhrip) [reply Readme.md]: Sure, no problem 👌

**Pull request merged**